### DISCUSSION TITLE
Show preview of image when viewing asset

### DISCUSSION THREAD

**Discussion by litkej** [atrocore/atrocore#287] [Q&A]
Hello - is there any possibility of viewing an image (or any asset that is a media type) within the page vs. having to download to open?

**Comment by atrolex** [comment]
What exactly do you mean? can you provide an example?

**Reply by litkej** [reply]
When looking at an asset (see image) - would like to be able to see a preview of the image within the page. 

![image](https://user-images.githubusercontent.com/87543116/153451810-2736b33c-89f0-4cf8-a0c4-158b572a4ac8.png)

**Reply by litkej** [reply]
The only option available to view it from what I see is to download the asset.

**Reply by atrolex** [reply]
I see, preview for TIFF is on the roadmap. Previews for certain file types are already available.

**Reply by litkej** [reply]
Is there a listing of which file types are enabled?

**Reply by atrolex** [reply]
no

**Reply by rmoecklideveloper** [reply]
TIFF and PSD Preview would be much appreciated. Is it still on the way? In my opinion the idea of a DAM/MAM is to store an asset in it's best original quality and that is often tiff oder psd.

**Reply by atrolex** [reply]
It will be implemented someday. For now it is low prio.
